### PR TITLE
chore: next meeting date and time

### PR DESCRIPTION
**Pull request by avishnu**

### CODE REVIEW

**Review by niladrih — APPROVED** [review]

**Review by Abhinandan-Purkait — APPROVED** [review]

File: README.md
Comment (tiagolobocastro) [inline README.md:14]: That is on the invite, but I put the wrong link above, should be:
https://calendar.app.google/TBQthr3oW42uyJqz5

**Review by tiagolobocastro — APPROVED** [review]

File: README.md
Comment (avishnu) [reply README.md]: Resolved with the latest commit.

**Comment by avishnu** [comment]
Got approval from 3 out of 4 maintainers. Merging now.
@orville-wright kindly take a look retrospectively.

**Comment by orville-wright** [comment]
@avishnu please use the Maintainers Group & chat https://groups.google.com/g/openebs-team .... for all Maintainers discussions.
-That is what we created it for.  - It is especially important where a decisions need to be made. 

I don't see any announcement to the Maintainers Group relating to this PR ? - and I cant tell what this PR is for ?  - other than a small change the readme.
https://groups.google.com/g/openebs-team

**Comment by avishnu** [comment]
> @avishnu please use the Maintainers Group & chat https://groups.google.com/g/openebs-team .... for all Maintainers discussions. -That is what we created it for. - It is especially important where a decisions need to be made.
> 
> I don't see any announcement to the Maintainers Group relating to this PR ? - and I cant tell what this PR is for ? - other than a small change the readme. https://groups.google.com/g/openebs-team

This PR is to add the calendar link for community users to know the schedule for the next meeting. Hope that suffices.

**Comment by orville-wright** [comment]
I see it. I kind of missed that as it looked a bit like the 2 google links were the same. But now I see they are different.

**Pull request merged**
